internal/observability/graph.go

```diff
@@ -37,57 +37,57 @@ var (
 	// Latency buckets for histograms. At some point, we may want to make these
 	// configurable.
 	_ms      = time.Millisecond
-	_buckets = []time.Duration{
-		1 * _ms,
-		2 * _ms,
-		3 * _ms,
-		4 * _ms,
-		5 * _ms,
-		6 * _ms,
-		7 * _ms,
-		8 * _ms,
-		9 * _ms,
-		10 * _ms,
-		12 * _ms,
-		14 * _ms,
-		16 * _ms,
-		18 * _ms,
-		20 * _ms,
-		25 * _ms,
-		30 * _ms,
-		35 * _ms,
-		40 * _ms,
-		45 * _ms,
-		50 * _ms,
-		60 * _ms,
-		70 * _ms,
-		80 * _ms,
-		90 * _ms,
-		100 * _ms,
-		120 * _ms,
-		140 * _ms,
-		160 * _ms,
-		180 * _ms,
-		200 * _ms,
-		250 * _ms,
-		300 * _ms,
-		350 * _ms,
-		400 * _ms,
-		450 * _ms,
-		500 * _ms,
-		600 * _ms,
-		700 * _ms,
-		800 * _ms,
-		900 * _ms,
-		1000 * _ms,
-		1500 * _ms,
-		2000 * _ms,
-		2500 * _ms,
-		3000 * _ms,
-		4000 * _ms,
-		5000 * _ms,
-		7500 * _ms,
-		10000 * _ms,
+	_buckets = []int64{
```

Comment:
nit: now that we're losing the unit information from the type, it may be prudent to rename this to `_bucketsMS`.